enterprise/management/commands/migrate_enterprise_catalogs.py

```diff
@@ -40,7 +46,14 @@ def handle(self, *args, **options):
             raise CommandError(_('A user with the username {username} was not found.').format(username=api_username))
 
         client = EnterpriseCatalogApiClient(user=user)
-        for enterprise_catalog in EnterpriseCustomerCatalog.objects.all():
+
+        catalog_uuid = options.get('catalog_uuid')
+        if catalog_uuid:
+            queryset = EnterpriseCustomerCatalog.objects.filter(uuid=catalog_uuid)
```

Comment:
should this support one uuid or potentially a list?

Reply:
I assumed if they were running it on the shell they could do the for-loop in bash, but happy to update this. should be easy